src/redux/bookApi.js

```diff
@@ -16,6 +16,7 @@ const createBooksList = (obj) => {
 export const fetchBooks = async () => {
   try {
     const res = await axios.get(baseUrl);
+
     return createBooksList(res.data);
   } catch (err) {
     throw new Error(err);
```

Comment:
- [ ] [OPTIONAL] For best practices, kindly remove any console.log in your code

![Screenshot_20220716_095934](https://user-images.githubusercontent.com/25227022/179348301-79a66e97-3772-48a5-854e-0152ac376689.png)